_dashboards/discover/visualizing-ad.md

```diff
@@ -0,0 +1,158 @@
+
+Alerts and anomalies are visualized in line charts on a dashboard. To create an alerting monitor using the Discover application and sample data and to visualize the alerts on a dashboard, follow these steps:  
+
+1. From the OpenSearch Dashboards main menu, select **Discover**.
+1. From **<window>**, select the **<name>** sample dataset. You will see a pre-populated dashboard with line chart visualizations.
+1. Select the ellipsis icon from the **<name>** visualization pane, and then from the **Options** pop-up window, choose **Alerting** > **Add alerting monitor**.
+1. Select **Create new monitor**.
+1. From the **Add alerting monitor** window, define the metrics and set the thresholds under **Monitor details** and **Triggers**.
+1. Under **Monitor details**, select <option> from the dropdown menu.
+1. Under **Triggers**, select <option> from the dropdown menu, and then specify a name and severity level for the trigger.
+1. _Optional_: In **Notifications**, select <option> from the dropdown menu, and then customize the notification message and select the Notification channel.
+1. Choose **Create monitor**.
+
+Once you have created a new monitor, the monitor is added to the visualization, as shown in the following image.  
+
+<insert UI>
+
+## Adding an associated monitor
+
+You can add existing monitors to a visualization using the Discover application instead of Stack Management, giving you a single user interface where can add, view, and edit monitors and monitor details. Continuing with the visualization and dashboard in the preceding tutorial, follow these steps to associate an existing monitor to a visualization:
+
+1. From the dashboard, select the ellipsis icon on the **<name>**, then **Alerting**.
+1. Select **Associate existing monitor**.
+1. From the **Select monitor to associate** dropdown menu, view the list of existing monitors and then select the desired monitor. In this example, select **<name>**. Note that basic information about the monitor is summarized in the window. To view more comprehensive details, select **View monitor page**, which opens the monitor details from the Alerting plugin page. 
+1. Verify you have selected the appropriate monitor, and then select **Associate monitor**. 
+
+An existing monitor is now associated to the visualization, as shown in the following image:
+
+<insert UI>
+
+## Exploring alerting monitor details
+
+Once you have created or associated alerting monitors, you can verify the monitor is generating the alerts and explore the alert details by following these steps:
+
+1. From the dashboard, view the <**name**> visualization. Alerts are indicated with a red triangle. 
+1. Hover over a triangle to view the alert count details. To investigate the alert details, select a triangle to activate the pop-up window containing the monitor details. Alternatively, select the ellipsis icon in the visualization pane and choose **View events** to view those details.
```

Comment:
pop-up -> flyout may be better wording? And within the view events flyout, I probably wouldn't call them "monitor details" - rather, its a more fine-grained breakdown of all of the events for each plugin resource (monitor). To really see more details about a particular monitor, users can click on the monitor name within this flyout to get full details on the respective plugin details page.